blivet/autopart.py

```diff
@@ -494,15 +486,17 @@ def do_autopart(storage, data, min_luks_entropy=0):
         raise NoDisksError(_("No usable disks selected"))
 
     disks = _get_candidate_disks(storage)
-    devs = _schedule_implicit_partitions(storage, disks, min_luks_entropy)
+    devs = _schedule_implicit_partitions(storage, disks)
     log.debug("candidate disks: %s", disks)
     log.debug("devs: %s", devs)
 
     if disks == []:
         raise NotEnoughFreeSpaceError(_("Not enough free space on disks for "
                                         "automatic partitioning"))
+    if min_luks_entropy is not None:
+        luks_data.min_entropy = min_luks_entropy
```

Comment:
I think this needs to happen before ``_schedule_implicit_partitions()`` is called.